packages/app/src/app/pages/Live/BlinkingDot.js

```diff
@@ -1,37 +1,33 @@
-import React from 'react';
-import styled from 'styled-components';
+import React, { useEffect, useState } from 'react';
 import RecordIcon from 'react-icons/lib/md/fiber-manual-record';
+import styled from 'styled-components';
 
 const DotContainer = styled.div`
-  font-size: 4rem;
-  display: block;
   color: rgb(253, 36, 57);
+  display: block;
+  font-size: 4rem;
 
   svg {
     transition: 0.3s ease opacity;
   }
 `;
 
-export default class BlinkingDot extends React.PureComponent {
-  state = {
-    showing: true,
-  };
+const BlinkingDot = () => {
+  const [showing, setShowing] = useState(true);
```

Comment:
`isVisible/visible` (adjective) instead of `showing` (verb) sound more natural for the state description.

Reply:
I just kept the names that were already there to make it easier to reason about when looking at the refactor